easybuild/easyconfigs/h/HPCX/HPCX_UCX-1.5.0rc1-foss-2018b.eb

```diff
@@ -0,0 +1,42 @@
+# Note:
+# This is an easyconfig file for EasyBuild, see https://github.com/easybuilders/easybuild
+easyblock = 'ConfigureMake'
+
+name = 'HPCX_UCX'
+version = '1.5.0rc1'
+
+homepage = 'http://www.openucx.org/'
+description = """Unified Communication X
+An open-source production grade communication framework for data centric
+and high-performance applications
+"""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
```

Comment:
UCX belongs at the GCCcore level, and it's better to use official releases of it. See https://github.com/easybuilders/easybuild-easyconfigs/pull/7924